tests/test_errors.py

```diff
@@ -160,96 +100,96 @@ def test_unsupported_function_call() -> None:
 
 
 def test_data_error() -> None:
-    with pytest.raises(DataError):
-        raise DataError()
+    with pytest.raises(errors.DataError):
+        raise errors.DataError()
```

Comment:
These tests feel like `errors.DataError is errors.DataError` but I don't know how to better test them.

Reply:
Jut makes sure they exist and are in the right place.